src/monitoring_tests/reference_solver_surplus_test.py

```diff
@@ -52,13 +52,16 @@ def compare_orders_surplus(
                 10**36,
             )
 
-            trade_alt = self.get_trade_alternative(uid, auction_instance)
-            if trade_alt is None:
-                self.logger.warning(
-                    f"No alternative trade for uid {uid} and "
+            ref_solver_response = self.solve_order_with_reference_solver(
```

Comment:
Potentially rename this to `solution_alt`? But this might be confusing in the sense that the solution format is then different in `solution` and `solution_alt`. Otherwise we could also change the variable name in `get_trade_alternative`, which is currently `solution`.

Reply:
I would probably say now that the `get_trade_alternative` name for the function might be not very good after all. The function is oblivious to whether this is an alternative execution or not, and the only thing it does is parse a given an execution and returns a trade.